hyperstyle/src/python/review/run_tool.py

```diff
@@ -151,6 +158,7 @@ def main() -> int:
             start_line = 1
 
         inspectors_config = {
+            "language": Language(args.language.upper()),
```

Comment:
Are you sure about this? Because by default this argument is None, so if you try to run `upper()` with None, you will get an error `AttributeError: 'NoneType' object has no attribute 'upper'`